### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by afti-githobo**
I think this is all the cleanup/commenting stuff I wanted to get done **except** for bringing the doc comments in pylons_wallet.dart up to date. That'll be happening tomorrow, as a separate PR. (There will be no code changes whatsoever there.) I'm requesting review here _now_ because I want to be certain I didn't break anything - in particular, refactoring the dev wallet implementation to inherit from the prod wallet implementation seemed correct, but I want Jawad to take a look over that so he can tell me if I broke anything.

### CODE REVIEW

File: lib/src/pylons_wallet_comm_util.dart [hunk 1]
```diff
@@ -192,27 +190,23 @@ class PylonsWalletCommUtil {
...
     }
-    return false;
   }
 
+  /// Parses the base64-encoded UTF8 string [response] and outputs a [Tuple2<String, List<String>>].
+  /// The first element of the tuple is the response's key; the second element is a list containing
+  /// the response data.
   static Tuple2<String, List<String>> procResponse(String response) {
     var decoder = const Utf8Decoder();
     var splut = response.split(',');
```
Comment (mustafapylons) [inline lib/src/pylons_wallet_comm_util.dart:213]: Let's use a more meaningful variable other than "splut".
Outside this fix, this is good clean code.

**Comment by kjawadDeveloper2** [comment]
@afti-githobo  if you can modify the old methods you write with the one i made . Can you do that? Like removing tuple with the sdk response.

File: lib/src/pylons_wallet_comm_util.dart (hunk 1 above)
Comment (MikeSofaer) [reply lib/src/pylons_wallet_comm_util.dart:213]: Is it not meaningful? Once it's been split, it's now splut. But perhaps too much of a pun.

**Comment by afti-githobo** [comment]
> 
> 
> @afti-githobo if you can modify the old methods you write with the one i made . Can you do that? Like removing tuple with the sdk response.

that's not within the scope of this pr, but i will do that today!

**Comment by afti-githobo** [comment]
actually, @kjawadDeveloper2, i don't really understand how the calls you implemented are supposed to work?

where are these being completed? like, can you show me how the logic flows? i'm not following it and i feel like i have to be missing something b/c what i'm reading isn't making sense to me

**Review by kjawadDeveloper1 — APPROVED** [review]
Looks good from myside.

**Comment by afti-githobo** [comment]
Superseded by #29.

**Pull request closed without merging**